03-Node-npm-modules/artist.js

```diff
@@ -0,0 +1,20 @@
+var Artist = function(name, awards){
+	this.name = name;
+	this.awards = awards || [];
+};
+
+Artist.prototype.recognice = function(){
+	var msg = this.name + " has the following awards: ";
+	for(i = 0; i < this.awards.length; i++){
+		msg += this.awards[i] + ' ';
```

Comment:
This is good, but you can archive the same with the javascript's method `.join()`. Example `var msg = this.name + " has the following awards: " + this.awards.join(' ')`